src/clients/QBittorrent.ts

```diff
@@ -117,9 +112,9 @@ export default class QBittorrent implements TorrentClient {
 			);
 		}
 
-		const cookieArray = response.headers.raw()["set-cookie"];
+		const cookieArray = response.headers.get("set-cookie");
```

Comment:
small style nit - now that we're accessing a `Headers` object with `.get`, I think we can use the proper casing (previously everything was lowercased in order to support case insensitivity)

```suggestion
		const cookieArray = response.headers.get("Set-Cookie");
```

Reply:
ah good catch. after reading up it seems like native fetch in node will not expose Set-Cookie headers to us directly in order to preserve compatibility with browser `fetch`.